docs/src/whatsnew/latest.rst

```diff
@@ -53,6 +53,10 @@ This document explains the changes made to Iris for this release
 #. `@pp-mo`_ corrected the use of mesh dimensions when saving with multiple
    meshes.  (:issue:`5908`, :pull:`6004`)
 
+#. `@trexfeathers`_ fixed the datum :class:`python:FutureWarning` to only be raised if
+   the ``datum_support`` :class:`~iris.Future` flag is disabled AND a datum is
+   present the loaded NetCDF grid mapping. (:issue:`5749`, :pull:`6050`)
```

Comment:
"AND a datum is present the loaded NetCDF grid mapping." I think this should be "AND a datum is present (ON) the loaded NetCDF grid mapping."

Reply:
```suggestion
   present on the loaded NetCDF grid mapping. (:issue:`5749`, :pull:`6050`)
```